automation/symbols-generation/symbolstore.py

```diff
@@ -403,7 +418,7 @@ def ProcessFileWork(self, file, arch_num, arch, vcs_root, dsymbundle=None):
                 # MODULE os cpu guid debug_file
                 (guid, debug_file) = (module_line.split())[3:5]
                 # strip off .pdb extensions, and append .sym
-                sym_file = re.sub("\.pdb$", "", debug_file) + ".sym"
+                sym_file = re.sub(".pdb$", "", debug_file) + ".sym"
```

Comment:
What's the purpose of this change? I think the previous line was correct; we want to turn "hello.pdb" into "hello", but leave "helloxpdb" as-is. In any case, it makes no difference to us here because libmegazord isn't used on Windows, and pdb files are only used on Windows.

Reply:
My Python linter reported "unsupported escape character in string".

After thinking about this more, I went the wrong way.  I should have added an extra `\` or maybe better changed this to a raw string.  Once this is merged, I'll open a followup to fix this.

Reply:
Oops, didn't realize that this wasn't approved yet.  I made that fix in this PR instead.